### ISSUE TITLE
use line-in feature for sound on android 9 below

### ISSUE THREAD

**Issue by GDjkhp** [rom1v/sndcpy#147]
Description:

i'd tested my phone for the audio jack lined-in on my pc, uhh ye just that simple

[Link to video here, sorry for quality](https://drive.google.com/file/d/11V_Go6y1q3RLl9ydljEDKB9XD4K18b0B/view?usp=sharing)

**Comment by mrfoxie** [comment]
I tried to install but it is showing this error
```
adb: failed to install .\sndcpy.apk: Failure [INSTALL_FAILED_OLDER_SDK: Failed parse during installPackageLI: /data/app/vmdl403410614.tmp/base.apk (at Binary XML file line #7): Requires newer sdk version #29 (current version is #27)]
```

**Comment by rom1v** [comment]
> Requires newer sdk version #29 (current version is #27)]

sndcpy requires Android 10.

**Comment by rom1v** [comment]
> i'd tested my phone for the audio jack lined-in on my pc, uhh ye just that simple

Yes, it's easier with a jack lined-in. But newer devices don't have jack plug.
